simtools/db_handler.py

```diff
@@ -109,3 +109,38 @@ def collectAllModelParameters(telescopeType, version):
     with open(_yamlFile, 'r') as stream:
         _allPars = yaml.load(stream, Loader=yaml.FullLoader)
     return _allPars
+
+
+def writeModelFile(fileName, destDir):
+    '''
+    Find the fileName in the model files location and write a copy
+    at the destDir directory.
+
+    Parameters
+    ----------
+    fileName: str or Path
+        File name to be found and copied.
+    destDir: str or Path
+        Path of the directory where the file will be written.
+    '''
+    destFile = Path(destDir).joinpath(fileName)
+    file = Path(getModelFile(fileName))
+    destFile.write_text(file.read_text())
+
+
+def getModelFile(fileName):
```

Comment:
More or less same comment as above.

Reply:
This is just needed for the writeModelFile, for the sake of organization and readbility.

Reply:
OK, so this will be replaced later by a function that gets a file from the DB I guess. There I have the option of reading the file and keeping it as data without writing it to disk, but I don't see a use case for that. So there's a chance the two functions will simply merge when we move to the DB.